src/components/SafeListSidebar/AddSafeButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import Fab from '@material-ui/core/Fab'
+import AddIcon from '@material-ui/icons/Add'
+import { OPEN_ADDRESS } from 'src/routes/routes'
+import { Text } from '@gnosis.pm/safe-react-components'
+
+interface Props {
+  onAdd: () => unknown
```

Comment:
why not void? we don't use the returned value anywhere

Reply:
Since this component doesn't use and doesn't care about the return value, the parent component may pass whatever function it wants. Including async functions. So no need to limit it to void.

Reply:
https://www.typescriptlang.org/play?#code/C4TwDgpgBAag9gSwCYDECuA7AxlAvFACgEo8A+KAN0SQCgas4MBnYS69bALlncx32JkoAFiA

🤔